+++
widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 10  # Order that this section will appear.

title = "Our research"


[design]
  # Choose how many columns the section has. Valid values: 1 or 2.
  columns = "1"

[design.background]
  # Apply a background color, gradient, or image.
  #   Uncomment (by removing `#`) an option to apply it.
  #   Choose a light or dark text color by setting `text_color_light`.
  #   Any HTML color name or Hex value is valid.

  # Background color.
  # color = "navy"

  # Background image.
  #image = "hero.jpg"  # Name of image in `static/img/`.
  #image_darken = 0.6  # Darken the image? Range 0-1 where 0 is transparent and 1 is opaque.

  # Text color (true=light or false=dark).
  text_color_light = false

[design.spacing]
  # Customize the section spacing. Order is top, right, bottom, left.
  padding = ["20px", "0", "20px", "0"]

[advanced]
 # Custom CSS.
 css_style = ""

 # CSS class.
 css_class = ""
+++

The genome provides a fundamental layer of information for cellular transitions in the form of coding and regulatory sequences. One conserved mode of regulation is conferred through RNA structure, which directly dictates RNA function. Recent studies have uncovered key roles for RNA structures in the regulation of many steps of gene expression, including transcription, splicing, translation, localization, and decay. Some RNA structures have been identified as master regulators of essential biological pathways, such as the GAIT elements modulating the inflammatory response and G-quadruplexes mediating translation and regulation in cancer. To perform their regulatory roles, RNA structures generally work in concert with protein factors. These include RNA helicases, conserved enzymes that are specialized in remodeling RNA structures, and whose malfunction in humans can lead to disease.

While the regulatory role of RNA structures in gene expression and cellular functions is unequivocal, many fundamental questions remain regarding the identification of regulatory RNA structural motifs, their molecular mechanisms and their biological outcomes, especially in vivo.


<!--
During Jean-Denis’ postdoctoral studies, he used molecular, cellular and RNA biology to study a fundamental process in all animals, the maternal-to-zygotic transition, where post-transcriptional regulation orchestrates a profound change in the embryonic transcriptional landscape (Fig MZT (see Figures folder)). Here, he asked two central questions: what are the RNA structures formed during early embryogenesis and how do they interact with cellular factors to control gene expression during development?

To address these questions, he developed innovative genome-wide in vivo RNA probing techniques to gain structural information on the zebrafish transcriptome during embryogenesis. In addition, he performed high throughput approaches to measure mRNA abundance and translation genome-wide in vivo. By integrating this information, he generated a global interaction map between RNA structure, mRNA stability and translation during early embryogenesis (Beaudoin JD*, Novoa EM* et al. Nat. Struct. Mol. Biol. 2018).

These results provide three major insights. First, contrary to the current dogma, I found that untranslated mRNAs are more structured in vivo compared to in vitro. Second, globally, translation guides mRNA structures rather than structure guiding translation. Third, RNA structures remodeled in 3’-UTRs are enriched for regulatory activity during development.

These findings have generated three main questions that will be the foundation of the research program in the Beaudoin lab: What are the cellular factors shaping RNA folding and activity in vivo (Area 1)? What are the cis-regulatory elements controlling translation during embryogenesis (Area 2)? What are the molecular mechanisms by which RNA helicases modify RNA structures to regulate gene expression during vertebrate development (Area 3)?
-->
